gitpod-io/gitpod
# [run-gp] experiment with adding run-gp layer to subset of users

## Description
This PR introduces a way to add a docker layer containing [run-gp](github.com/gitpod-io/run-gp) wrapped by a shell cmd `gp-run`. 

Co-Authored: @akosyakov @andreafalzetti @felladrin 

## Related Issue(s)

Fixes #

## How to test
1. Start a workspace and verify that `gp-run` is not available as a command
2. Add your email or userId in ConfigCat and run the test again
3. Start a workspace and check that `gp-run` is available and starts `run-gp`

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
